spec/bolt/cli_spec.rb

```diff
@@ -180,7 +181,7 @@ def stub_config(file_content = {})
       )
     end
 
-    it 'errors without a targeting option' do
+    it 'errors without a targetting option or plan-hierarchy' do
```

Comment:
```suggestion
    it 'errors without a targeting option or plan-hierarchy' do
```

Apparently 'targetting' is an obsolete / uncommon British English spelling. 😅

Reply:
🧐